src/backend/api/Fusion.Resources.Api/Controllers/ResponsibilityMatrix/Requests/UpdateResponsibilityMatrixRequest.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using FluentValidation;
+using Fusion.Resources.Domain.Commands;
+
+namespace Fusion.Resources.Api.Controllers
+{
+    public class UpdateResponsibilityMatrixRequest
```

Comment:
Create and Update are identical. Should we use the same request model to avoid duplication? 
Can be split if they suddenly differ for some reason.